src/storage/DataAccessorBasedStore.ts

```diff
@@ -382,18 +388,22 @@ export class DataAccessorBasedStore implements ResourceStore {
     // Solid, §5.3: "Servers MUST create intermediate containers and include corresponding containment triples
     // in container representations derived from the URI path component of PUT and PATCH requests."
     // https://solid.github.io/specification/protocol#writing-resources
-    const modified = [];
+    let changes: Record<string, RepresentationMetadata> = {};
     if (!this.identifierStrategy.isRootContainer(identifier) && !exists) {
-      const container = this.identifierStrategy.getParentContainer(identifier);
+      const parent = this.identifierStrategy.getParentContainer(identifier);
       if (!createContainers) {
-        modified.push(container);
+        changes[parent.path] = new RepresentationMetadata(parent).add(SOLID_AS.terms.Activity, AS.Update);
       } else {
-        const created = await this.createRecursiveContainers(container);
-        modified.push(...created.length === 0 ? [ container ] : created);
+        const createdContainers = await this.createRecursiveContainers(parent);
```

Comment:
Any reason both the `created` and `container` parameters were renamed?

Reply:
Was wondering if there was a specific reason. The problem with changing variable names is that it changes the commit history of lines when using git tooling while the commit didn't actually impact that line. It's not that much of an issue now since only a few lines get impacted so I'm OK with it staying like this.

Reply:
Will keep that in mind next time 👍🏻